src/components/HeaderButtons.jsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+import toast from "react-hot-toast";
+import { login, logout } from "@/lib/session";
+
+function handleLogout(router) {
+  logout();
+  toast.success("Logout success!");
+  router.push("/");
+}
```

Comment:
put this inside the HeaderButtons component (line 24) so that you dont have to pass the `router`

Reply:
Resolved in [f6a1822](https://github.com/alfonsusac/devscale-eventmaker/pull/70/commits/f6a18225df093cf94f178c73faae084fca339aa2)

Reply:
- closes: #73